en/docs/apis/create-and-publish-your-first-api.md

```diff
@@ -21,27 +17,27 @@ Follow this procedure to create an API:
     | **API Version**  | `1.0.0`                                          |
     | **Endpoint**     | `https://disease.sh`|
 
-5. Click **Create**. This displays an overview of the API.
+5. Click **Create**. This creates the API and displays an overview of the API.
 
-    ![API overview](../assets/img/apis/api-overview.png){.cInlineImage-full}
+    ![API overview](../assets/img/apis/api-overview-page.png){.cInlineImage-full}
        
-6. Under **API Configuration**, click **Resources**. Then follow this procedure to add a resource to the API:
-    1. Select `GET` as the **HTTP verb**.
-    2. Enter `/v3/covid-19/states` as the **URI pattern**.
-    3. Click the **+** icon to add the resource.
-    4. Click **Save**. You can see a confirmation message to proceed with the changes.
-    5. Click **Save** to proceed.
+6. To add a resource to the API, follow this procedure:
+    1. Under **API Configuration**, click **Resources**.
+    2. Select `GET` as the **HTTP verb** drop-down list.
```

Comment:
Suggestion: "Select **GET** from the **HTTP Verb** drop-down list."

"as" should change to "from". Suggesting having "GET" in bold because drop-down list values are UI elements and not typed in. The 'V" in "Verb" is capital.